### ISSUE TITLE
Could you add a C# language binding

### ISSUE THREAD

**Issue by xbin9679-lab** [CrispStrobe/CrispASR#193]
Could you add a C# language binding?

**Comment by CrispStrobe** [comment]
did that. but most probably some thing was missed here or there. so would be very helpful if you could test this.

**Comment by CrispStrobe** [comment]
Implemented — C# binding lives at `bindings/csharp/` with `CrispASR.sln`, NativeMethods P/Invoke wrappers, Session class, and test project. Includes raw CTC logits accessor (f3ff5496). User was asked to test; no further feedback.
